src/test/convert-to-single-host.ts

```diff
@@ -8,10 +8,11 @@ import * as helpers from 'yeoman-test';
 import { readManifestFile } from "office-addin-manifest";
 import * as path from 'path';
 import { promisify } from "util";
-const readFileAsync = promisify(fs.readFile);
 const hosts = ["excel", "onenote", "outlook", "powerpoint", "project", "word"];
 const manifestFile = "manifest.xml";
 const packageJsonFile = "package.json";
+const prerelease = process.argv.indexOf('--prerelease') > -1;
```

Comment:
I was expecting that you would just add " || true" to the end to hardcode for now, rather than the changes below for each helpers.run() call.

Reply:
oh  I see